tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -118,11 +155,7 @@ public function userHasBeenCreated(string $user, string $displayName = null):voi
 		if ($displayName !== null) {
 			$userAttributes['displayName'] = $displayName;
 		}
-
-		$this->response = $this->sendOCSRequest(
-			'/cloud/users', 'POST', $this->getAdminUsername(), $userAttributes
-		);
-		$this->theHttpStatusCodeShouldBe(200);
+		$this->createUserWithRetry($user, $userAttributes);
```

Comment:
```suggestion
		$this->createUserWithRetry($user, $userAttributes);
		$this->theHttpStatusCodeShouldBe(200);
```

don't we need to check the status code in `given` statement

Reply:
its checked in the function itself since we need to retry it its checked inside rather than with an explicit function.